### ISSUE TITLE
upgrade.php errors

### ISSUE THREAD

**Issue by illori** [SimpleMachines/SMF#5987]
#### Description
 * Updating Settings.php...
Warning: preg_match_all(): Compilation failed: unknown property name after \P or \p at offset 122 in C:\wamp64\www\github2.1\Sources\Subs-Admin.php on line 1280

Warning: preg_replace(): Compilation failed: unknown property name after \P or \p at offset 122 in C:\wamp64\www\github2.1\Sources\Subs-Admin.php on line 1369

Warning: preg_match(): Compilation failed: unknown property name after \P or \p at offset 122 in C:\wamp64\www\github2.1\Sources\Subs-Admin.php on line 1600
 Successful.

 Successful.

Warning: preg_match_all(): Compilation failed: unknown property name after \P or \p at offset 122 in C:\wamp64\www\github2.1\Sources\Subs-Admin.php on line 1280

### Steps to reproduce
1. updated to latest on github as of a few minutes ago
2. run upgrade.php via CLI

### Environment (complete as necessary)
- Version/Git revision:
- Database Type: mariaDB
- Database Version:
- PHP Version: 7.1.33

### Additional information/references

**Comment by sbulen** [comment]
I could not reproduce this.  Made a few attempts, tried various versions of php, used the settings file provided in #5988 , CLI 7 non-CLI, etc...  

I think @jdarwood007 saw a similar message at one point earlier?   Any thoughts - have you seen this?

**Comment by jdarwood007** [comment]
@Sesquipedalian Seems related to the recent merge?

**Comment by sbulen** [comment]
These issues look like some issues reported about two weeks ago in the middle of testing of #5961 .  

I cannot reproduce these today, & I've tried...

My thinking at the moment is that this & #5988 are due to Illori's git & settings.php both being a little wonky.  She has a nifty script that updates git each evening & then automates the upgrade.  It does it by doing a pull, & a merge commit - so her current state is not matching the source on github.  I am pretty sure that got tangled up somehow, & subsequent merge commits weren't really bringing things in sync.  

I think we got her git repo proper today.  Need to work on the db & settings.php next.  I want to see what happens once get a clean DB, settings file & current git...
